mobile/pubspec.lock

```diff
@@ -129,16 +146,18 @@ packages:
     dependency: "direct dev"
     description:
       name: flutter_lints
-      url: "https://pub.dartlang.org"
+      sha256: aeb0b80a8b3709709c9cc496cdc027c5b3216796bc0af0ce1007eaf24464fd4c
+      url: "https://pub.dev"
     source: hosted
     version: "2.0.1"
   flutter_rust_bridge:
     dependency: "direct main"
     description:
       name: flutter_rust_bridge
-      url: "https://pub.dartlang.org"
+      sha256: "5aea0f3980dcd314f1890ef0d2392263817899cc15e543734b5d4dbe66b761eb"
+      url: "https://pub.dev"
     source: hosted
-    version: "1.61.1"
+    version: "1.62.0"
```

Comment:
🤔 It's strange that this is shown as diff. The main branch is already locked on 1.62.0.

https://github.com/get10101/10101/blob/16cf62e9bc073fdb78f7521c174d603c8437e33d/mobile/pubspec.lock#L153-L160